ocfweb/docs/docs/services/lab/lab-reservation-policy.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Lab Reservation Policy"]]
+
```

Comment:
Should add an annotation here describing who established the policy (the Board of Directors, in this case) and when. This lets us know who is allowed to change the policy in the future.